src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import CstgAddDomainDialog from './CstgAddDomainDialog';
+
+const meta: Meta<typeof CstgAddDomainDialog> = {
+  title: 'CSTG/CstgAddDomainDialog',
+  component: CstgAddDomainDialog,
+};
+export default meta;
+
+type Story = StoryObj<typeof CstgAddDomainDialog>;
+
+export const Default: Story = {
+  args: {
+    onAddDomains: (newDomain) => Promise.resolve(console.log('New domain added: ', newDomain)),
+  },
+};
```

Comment:
With the new dialog format, the dialog is automatically open in storybook now instead of pressing the on button.  Submitting still works and prints in the console the new domain, but we may need to update later if we would rather have the "Open" button displayed first and the user then clicks that.  Since nothing is broken though, I vote this is addressed in the ticket that fixes up the dialog pattern altogether.